views/productform.blade.php

```diff
@@ -317,7 +317,7 @@ class="custom-control custom-select"
 			'label' => 'Default due days after freezing',
 			'min' => -1,
 			'value' => $value,
-			'hint' => $__t('On moving this product to a freezer location (so when freezing it), the due date will be replaced by today + this amount of days') . ' (' . $__t('-1 means that this product will be never overdue') . ')'
+			'hint' => $__t('On moving this product to a freezer location (so when freezing it), the due date will be replaced by today + this amount of days') . ' (' . $__t('-1 means that this product will be never overdue') . '; ' . $__t('0 means that the due date will not be replaced') . ')'
```

Comment:
Concatenating two separate localization strings at the same place doesn't make sense, if then simply change the existing one.

However, consistency is also key, so then do that also for the description of "Default due days after thawing" (#1880 was about unification of those two product options) or for neither of them